spec/controllers/users_controller_spec.rb

```diff
@@ -340,36 +340,4 @@
       expect(response).to be_success
     end
   end
-
-  describe 'GET #notifications' do
-    context 'given a logged in useer' do
-      it 'should show notifications settings page successfully' do
-        sign_in @user
-        get :notifications, params: { id: @user.to_param }
-        expect(response).to be_success
-      end
-    end
-    context 'given no logged in user' do
-      it 'should deny access' do
-        get :notifications, params: { id: @user.to_param }
-        expect(response).to be_unauthorized
-      end
-    end
-  end
-
-  describe 'PATCH #update_notifications' do
-    it 'should update notifications settings successfully' do
```

Comment:
Where did this test go?

Reply:
Those controller actions aren't existing anymore, since the updating logic is handled by the default #update action.

Reply:
after having an intensive chat with @jonasnoki about just that, i am joining in on the question whether this should not be tested

Reply:
@jonasnoki Yeah makes sense... I'll adjust it in a minute.